src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int ARRAY_SIZE = 6;
+
+    public static void main(String[] args) {
+        Figure[] arrayOfFigure = new Figure[ARRAY_SIZE];
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (int i = 0; i < arrayOfFigure.length; i++) {
+            if (i < arrayOfFigure.length / 2) {
+                arrayOfFigure[i] = figureSupplier.getRandomFigure();
+            } else {
+                arrayOfFigure[i] = figureSupplier.getDefaultFigure();
+            }
+
+            arrayOfFigure[i].draw();
+        }
```

Comment:
You should create several random Figures, so you will use a loop. Please don't create a `new FigureSupplier()` inside the loop. Although you have done it correctly by placing `new FigureSupplier()` outside the loop, it's important to remember this practice for future coding.